src/message_interface/output_builder.rs

```diff
@@ -0,0 +1,304 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_client::{
+    bee_block::{
+        output::{
+            dto::{NativeTokenDto, OutputDto},
+            feature::dto::FeatureDto,
+            unlock_condition::dto::UnlockConditionDto,
+            AliasId, AliasOutputBuilder, BasicOutputBuilder, Feature, FoundryOutputBuilder, NativeToken, NftId,
+            NftOutputBuilder, TokenScheme, UnlockCondition,
+        },
+        DtoError,
+    },
+    Client,
+};
+
+use crate::Result;
+
+#[allow(clippy::too_many_arguments)]
+pub(crate) async fn build_basic_output(
+    client: Client,
+    // If not provided, minimum storage deposit will be used
+    amount: Option<String>,
+    native_tokens: Option<Vec<NativeTokenDto>>,
```

Comment:
Why is native_tokens included but NFTs not?

Reply:
Because NFTs are an own output type and native tokens not